README.md

```diff
@@ -16,3 +16,4 @@ decentralized and appless.
 ## Getting Started
 
 ## License
+Copyright 2017 - 2018 Lunchclass. and the Absolute project authors. Licensed under Apache License v2.0.
```

Comment:
I think '2017-2018' looks strange. Can we change this to 'since 2017'?
Also, this should be Absolute project authors rather than Lunchclass.
Some contributors can be absolute project authors but they may not be lunch class members.
What do you think @nadongguri?